akvo/rsr/models/indicator.py

```diff
@@ -778,6 +802,13 @@ def clean(self):
                                                     u'data to it'))
             raise ValidationError(validation_errors)
 
+        # Don't allow a data update to an aggregated parent period with 'percentage' as measurement
+        if self.period.indicator.children_aggregate_percentage():
```

Comment:
Raises a TypeError, I think it should be `self.period.indicator.children_aggregate_percentage` since the `@property` decorator is used on the method.